app/components/workflow-basics.js

```diff
@@ -197,7 +197,20 @@ export default Component.extend({
     /** Sets the selected journal for the current publication.
      * @param journal {DS.Model} The journal
      */
-    selectJournal(journal) {
+    async selectJournal(journal) {
+      let doiInfo = this.get('doiInfo');
+      doiInfo = {
+        'journal-title': journal.get('journalName'),
+        ISSN: journal.get('issns')
+      };
+
+      const nlmtaDump = await this.getNlmtaFromIssn(doiInfo);
+      if (nlmtaDump) {
+        doiInfo.nlmta = nlmtaDump.nlmta;
+        doiInfo['issn-map'] = nlmtaDump.map;
```

Comment:
This is getting shown to the user as `[object, object]` should be stringified or not displayed. 
![screen shot 2018-06-25 at 12 07 43 pm](https://user-images.githubusercontent.com/17414914/41861933-8d503c40-7870-11e8-87f2-dce452d2a326.png)

Reply:
This will also need to be updated in the submission/detail route

Reply:
After **those fixs** this PR is good to go

Reply:
my vote is don't show it! we already have ISSN showing